general-concepts/dependencies/text.xml

```diff
@@ -209,28 +209,46 @@ asterisk. Also note that when selecting all versions in a specific
 <body>
 
 <p>
-Sometimes two packages cannot be installed in parallel. This is handled by
-blockers. A softblocker is specified as follows:
+When two packages (package slots, versions) can not be installed
+simultaneously, blockers can be used to expose such a conflict
+to the package manager.
 </p>
 
+<p>
+There are two kinds of blockers: soft blockers and hard blockers.
+</p>
+
+<p>A soft blocker is defined using the following syntax:</p>
+
 <codesample lang="ebuild">
 RDEPEND="!app-misc/foo"
 </codesample>
 
 <p>
-Portage will try to resolve this conflict automatically if possible.
-Sometimes we need to use a hardblocker to ensure correct emerge order.
-Those cannot be resolved by Portage and must be taken care of by the user.
-A hardblocker is specified as follows:
+The package manager will try to resolve this conflict automatically.
+The package blocked by a soft blocker can be uninstalled <e>after</e>
+installing the package blocking it. However, it exempts the common
+files from file collision checks. Soft blockers are usually used
+to solve file collisions between packages and are meaningful only
+in <c>RDEPEND</c>.
+</p>
+
+<p>
+If it is strictly necessary to resolve the blocker before the package
+is built (installed) or if automatic resolution is undesired,
+a hard blocker must be used instead. Hard blockers are not resolved
+automatically, and are expressed using the following syntax:
 </p>
 
 <codesample lang="ebuild">
 RDEPEND="!!app-misc/foo"
 </codesample>
 
 <p>
-Hardblockers always take precedence over softblockers and need at least EAPI=2.
-Also note that blockers are usually <e>runtime</e> rather than buildtime.
+EAPIs 0 and 1 do not define the distinction between soft and hard
+blocks. In those EAPIs, only the single exclamation mark syntax is
+allowed and it is undefined whether it is interpreted as a soft or hard
+block.
```

Comment:
I'd rather not add such long explanations about deprecated EAPIs. The previous wording that at least EAPI 2 is required was just fine.

Reply:
I was actually wondering if we need to mention EAPI at all. But sure, I can go for the old concept.

Reply:
Fixed.